woodwork/table_accessor.py

```diff
@@ -144,7 +144,13 @@ def init(self,
             if self._schema.time_index is not None:
                 self._sort_columns(already_sorted)
 
+        # Record the value of
```

Comment:
Comment needed?

Reply:
yeah, moved the place where we set `make_index` at your suggestion and added a more descriptive comment there